monitor/plugins/spark_mesos/plugin.py

```diff
@@ -112,12 +111,11 @@ def _publish_measurement(self, jobs):
 
             time.sleep(MONITORING_INTERVAL)
 
-
     def _get_elapsed_time(self, gmt_timestamp):
         try:
             local_tz = tzlocal.get_localzone()
 
-        except Exception as e:
```

Comment:
What is the main purpose of this modification? Can you please document this refactor in the description of this PR?

Reply:
When used "Exception as e", It's define a new variable "e" that is not be used.